okapi-core/src/main/resources/log4j2.properties

```diff
@@ -10,8 +10,27 @@ appenders = console
 
 appender.console.type = Console
 appender.console.name = STDOUT
-appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = %d{HH:mm:ss} %-5p %-20.20C{1} %m%n
+appender.console.layout.type = JSONLayout
+appender.console.layout.compact = true
+appender.console.layout.eventEol = true
+appender.console.layout.stacktraceAsString = true
+
+## Folio fields
+appender.console.layout.requestId.type = KeyValuePair
+appender.console.layout.requestId.key = requestId
+appender.console.layout.requestId.value = $${FolioLoggingContext:requestId}
+
+appender.console.layout.tenantId.type = KeyValuePair
+appender.console.layout.tenantId.key = tenantId
+appender.console.layout.tenantId.value = $${FolioLoggingContext:tenantId}
+
+appender.console.layout.userId.type = KeyValuePair
+appender.console.layout.userId.key = userId
+appender.console.layout.userId.value = $${FolioLoggingContext:userId}
+
+appender.console.layout.moduleId.type = KeyValuePair
+appender.console.layout.moduleId.key = moduleId
+appender.console.layout.moduleId.value = $${FolioLoggingContext:moduleId}
```

Comment:
In https://stackoverflow.com/questions/31342950 they advise not to use camel case for plugin names.

Reply:
I haven't faced such issues. Maybe they are for old version of log4j2. Are you proposing to lowercase the variables?

Reply:
It is easier to simply lowercase requestId, tenantId, userId, moduleId and FolioLoggingContext than investigate whether that bug may affect us.

Reply:
done